src/Command/Remote/AliasesDownloadCommand.php

```diff
@@ -49,22 +50,25 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     $this->output->writeln(sprintf(
       'Cloud Platform Drush Aliases archive downloaded to <options=bold>%s</>',
```

Comment:
This message is extraneous except for debugging, and could actually be misleading because the archive is deleted before the command exits.